masterlist.yaml

```diff
@@ -34174,20 +34185,11 @@ plugins:
         itm: 4
         udr: 40
 
-  - name: 'ImpeREAL Empire Unique Cities - Sutch.esp'
-    url:
-      - link: 'https://www.nexusmods.com/oblivion/mods/37867/'
-        name: 'ImpeREAL Empire Unique Cities - Sutch'
-    dirty:
-      - <<: *quickClean
-        crc: 0xEB59AB31
-        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 2
-
   - name: 'Sutch Coblized.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/52617/'
         name: 'Sutch Coblized'
+    group: *latePatchesGroup
```

Comment:
```suggestion
```

The **Late Patches** group predates me, but it seems to primarily be used for filter patches & a few other patches. This mod only adds references to a city added by another mod, so the chances of conflicts are already extremely low. They'd also presumably be conflicts like collisions with other mod-placed references which cannot be resolved with load order, so the default group seems appropriate here. Also, this should've been added in a separate PR as these are separate mods from separate pages (despite one being an add-on for another), although that won't be necessary anymore as the change is being rejected.

Reply:
If Sutch is removed from the cities group, than this change isn't even required at all. It was added to break a cycle caused by putting ImpeREAL Empire Unique Cities - Sutch into the cities group.